WebRoot/js/zimbraMail/abook/model/ZmContact.js

```diff
@@ -426,9 +426,12 @@ function(node, args) {
 			contact._parseTagNames(node.tn);
 		}
 		AjxUtil.hashUpdate(contact.attr, node._attrs);	// merge new attrs just in case we don't have them
-		contact.list = args.list || new ZmContactList(null);
-		contact._list = {};
-		contact._list[contact.list.id] = true;
+		var arglist = args.list || new ZmContactList(null);
```

Comment:
Can't we have something like following?

```
if(!contact.list) {
     contact.list = args.list || new ZmContactList(null);
}

contact._list = contact._list || {};
contact._list[contact.list.id] = true;
```

Reply:
as per now we can't have like this.
looking for any alternative and robust way to fix contact deletion